src/calva-fmt/src/format.ts

```diff
@@ -135,6 +137,21 @@ export async function formatPositionInfo(
   };
 }
 
+function _calculateFormatRange(
+  config: { 'format-depth'?: number },
+  cursor: LispTokenCursor,
+  index: number
+) {
+  const formatDepth = config?.['format-depth'] ?? _defaultFormatDepth(cursor);
+  return cursor.rangeForList(formatDepth) ?? cursor.rangeForCurrentForm(index);
+}
+
+function _defaultFormatDepth(cursor: LispTokenCursor) {
```

Comment:
I think `default` is confusing here. How about just `_formatDetpth`?

Reply:
sure, sounds reasonable!